app/src/style/module/_admin-page.scss

```diff
@@ -0,0 +1,27 @@
+.admin-nav {
```

Comment:
These files in `module` would be better in individual component/view folders than in includes